### ISSUE TITLE
ballots in lobby

### ISSUE THREAD

**Issue by kgersen** [kgersen/Allegiance#176]
<+Imago> extend the ballot functions outside of the game into lobby
<+Imago> first thing i would do is do an end of game MVP vote for each team
<+Imago> #mvp Imago
<+Imago> #mvp C101
<+Imago> (only 1 per team allowed)
<+Imago> you can vote up untill the next game
<+Imago> MVPs will be announced at the start of the next game
<+Imago> that will help ranking with new data (you cant vote for yourself) and also help speed up lag between games

 _Imported from trac ticket [#177](https://trac.alleg.net//ticket/177),  created by Imago on 07-25-2010 at 13:10, last modified: 11-25-2013 at 06:48_

**Comment by kgersen** [comment]
Could you add a way to mutiny a commander while in lobby? Mainly to stop the afk with command issues.

_Trac comment by anonymous on 07-26-2010 at 20:59_

**Comment by kgersen** [comment]
Main use would be mutiny commanders, yes.

_Trac comment by Spunky on 02-22-2012 at 05:13_

**Comment by kgersen** [comment]
In [1092/allegsvn]:

```
#CommitTicketReference repository="allegsvn" revision="1092"
Fix #177 - ballots in Lobby.

Stage checks on votes readjusted to allow mutiny in the lobby, and game won't start while any votes are pending.
On the client side, vote text and availability are exposed from teamscreen to mdl with HasVote and VoteText.
```

_Trac comment by Spunkmeyer on 08-12-2012 at 03:04_

**Comment by kgersen** [comment]
nice

_Trac comment by imago on 11-25-2013 at 06:48_
